src/main/java/com/gregtechceu/gtceu/api/machine/SimpleTieredMachine.java

```diff
@@ -394,7 +394,23 @@ protected static EditableUI<SlotWidget, SimpleTieredMachine> createBatterySlot()
             slotWidget.setHandlerSlot(machine.chargerInventory, 0);
             slotWidget.setCanPutItems(true);
             slotWidget.setCanTakeItems(true);
-            slotWidget.setHoverTooltips(LangHandler.getMultiLang("gtceu.gui.charger_slot.tooltip", GTValues.VNF[machine.getTier()], GTValues.VNF[machine.getTier()]).toArray(new MutableComponent[0]));
+            slotWidget.setHoverTooltips(LangHandler.getMultiLang("gtceu.gui.charger_slot.tooltip", GTValues.VNF[machine.getTier()], GTValues.VNF[machine.getTier()]).toArray(Component[]::new));
+        });
+    }
+
+    /**
```

Comment:
You may want to update that javadoc comment :D

Reply:
me when I copypaste the code: